functests/1_performance/performance.go

```diff
@@ -201,14 +201,14 @@ var _ = Describe("[rfe_id:27368][performance]", func() {
 			}
 		})
 		It("[test_id:42400][crit:medium][vendor:cnf-qe@redhat.com][level:acceptance] stalld daemon is running as sched_fifo", func() {
-                         for _, node := range workerRTNodes {
-                                 pid, err := nodes.ExecCommandOnNode([]string{"pgrep", "-f", "stalld"}, &node)
-                                 Expect(err).ToNot(HaveOccurred())
-                                 sched_tasks, _ := nodes.ExecCommandOnNode([]string{"chrt", "-ap", pid}, &node)
-				 Expect(sched_tasks).To(ContainSubstring("scheduling policy: SCHED_FIFO"))
-				 Expect(sched_tasks).To(ContainSubstring("scheduling priority: 10"))
-			 }
-                 })
+			for _, node := range workerRTNodes {
+				pid, err := nodes.ExecCommandOnNode([]string{"pgrep", "-f", "stalld"}, &node)
```

Comment:
can ever "pid" be empty string? IOW, are we guaranteed that if `err == nil` THEN the "pid" is valid?

Reply:
after checking this , if I search for non existed service pid :
 automatically : it will exit with status 1 error & the test will fail 
 manually : I redirected the stderr to another file but found nothing 
 So I added a check for the pid to not to be empty after checking the error.